src/controllers/expenses.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const { filterExpanses } = require('../utils/filterExpanses.js');
```

Comment:
Typo in the imported function name 'filterExpanses'. It should be 'filterExpenses'.